Symfony/CS/Fixer/PSR2/UnneededControlParenthesesFixer.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\PSR2;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class UnneededControlParenthesesFixer extends AbstractFixer
+{
+    /**
+     * To be removed when PHP support will be 5.5+.
+     *
+     * @var array|null
+     */
+    private static $controlStatements = null;
+
+    private static $loops = array(
+        'switch_case' => array('lookupTokens' => T_CASE, 'neededSuccessors' => array(';', ':')),
+        'echo_print' => array('lookupTokens' => array(T_ECHO, T_PRINT), 'neededSuccessors' => array(';', array(T_CLOSE_TAG))),
+        'return' => array('lookupTokens' => T_RETURN, 'neededSuccessors' => array(';')),
+        'clone' => array('lookupTokens' => T_CLONE, 'neededSuccessors' => array(';', ':', ',', ')')),
+    );
+
+    /**
+     * Dynamic yield option set on constructor.
+     */
+    public function __construct()
+    {
+        // To be moved back on static when PHP support will be 5.5+
+        if (defined('T_YIELD')) {
+            self::$loops['yield'] = array('lookupTokens' => T_YIELD, 'neededSuccessors' => array(';', ')'));
+        }
+    }
+
+    /**
+     * @param array|null $controlStatements
+     */
+    public static function configure(array $controlStatements = null)
+    {
+        self::$controlStatements = $controlStatements;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        // Checks if specific statements are set and uses them in this case.
+        $loops = empty(self::$controlStatements) ? self::$loops : array_intersect_key(self::$loops, array_flip(self::$controlStatements));
```

Comment:
empty array shouldn't run all fixers

consider a case when your configure array is build dynamically and in some cases it computed as empty array to not fix anything